templates/docker-compose.yml.j2

```diff
@@ -12,6 +12,12 @@ services:
       - /var/log:/mnt/log:ro
     {%- endif %}
 
+    {%- if beat == 'auditbeat' %}
+    cap_add:
+      - AUDIT_CONTROL
+    pid: "host"
```

Comment:
For better or worse, we don't explicitly quote the other strings in this YAML, so let's not here.

Reply:
gotcha. fixed in bd91cca